yarn.lock

```diff
@@ -21447,7 +21447,7 @@ type-of@^2.0.1:
   version "2.0.1"
   resolved "https://registry.yarnpkg.com/type-of/-/type-of-2.0.1.tgz#e72a1741896568e9f628378d816d6912f7f23972"
 
-typedarray-to-buffer@^3.1.5, typedarray-to-buffer@~3.1.5:
+typedarray-to-buffer@~3.1.5:
```

Comment:
Please drop changes in this file. I think they don't need to be part of this commit.

Reply:
Sure, I just checked out the repo and run `yarn`, it wasn't intentional.